src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -830,3 +831,60 @@ def create_or_update_service_hooks_for_sentry_app(
                 events=events,
                 webhook_url=webhook_url,
             )
+
+
+@instrumented_task(
+    "sentry.sentry_apps.tasks.sentry_apps.regenerate_service_hooks_for_installation",
+    taskworker_config=TaskworkerConfig(
+        namespace=sentryapp_control_tasks, retry=Retry(times=3), processing_deadline_duration=60
+    ),
+    **CONTROL_TASK_OPTIONS,
+)
+def regenerate_service_hooks_for_installation(
```

Comment:
1:1 copied the existing [`create_or_update_service_hooks_for_installation`](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/sentry_apps/service_hook_manager.py#L5) into a task, we'll need to cutover the existing usage of `create_or_update_service_hooks_for_installation` so that will take a few more PRs